Issue#3

josezindia opened the pull request:
gallery submission dashboard page

Commit:
Gallery submissions

Commit:
Gallery submissions

Commit:
Gallery submissions

Commit:
Gallery submissions

Commit:
gallery submission dashboard page

Commit:
gallery submission dashboard page

Commit:
gallery submission dashboard page

Commit:
Merge branch 'issue#3' of https://github.com/BCStudentSoftwareDevTeam/Ullman_Galleries into issue#3

Jessonsotoventura commented on app/templates/views/gallery_view.html:
```diff
@@ -0,0 +1,44 @@
+{% block title %}Submissions{% endblock %}
+{% block body %}
+
+<div class="inner">
```
Where is this class coming from? Is it bootstrap?

Jessonsotoventura commented on app/templates/views/gallery_view.html:
```diff
@@ -0,0 +1,44 @@
+{% block body %}
+
+<div class="inner">
+        <h1 style="text-align:center;">Submissions</h1>
```
Please use bootstrap for styling when possible, instead of using style tag.
Bootstrap has a text-center class that does this same action

Jessonsotoventura commented on app/templates/views/gallery_view.html:
```diff
@@ -0,0 +1,44 @@
+
+<div class="inner">
+        <h1 style="text-align:center;">Submissions</h1>
+          <hr class="intro-divider">
```
Where is this class coming from?

Jessonsotoventura commented on app/templates/views/gallery_view.html:
```diff
@@ -0,0 +1,44 @@
+          <div class="center">
+            <table class="table table-striped table-bordered">
+   <thead>
+    <tr style="background-color: #004154; color: white;">
```
Please leave color alone, we will modify it using a theme throughout the system

Jessonsotoventura commented on app/templates/views/gallery_view.html:
```diff
@@ -0,0 +1,44 @@
+   <thead>
+    <tr style="background-color: #004154; color: white;">
+       
+      <th width="10%">Name </th>
```
Please use bootstrap grid system to instead of the width modifier
https://getbootstrap.com/docs/4.0/layout/grid/

Commit:
Working on issue #3

Commit:
Working on issue #3

Commit:
Fixing conflict

Commit:
Merged with development

Commit:
Merge branch 'issue#3' of github.com:BCStudentSoftwareDevTeam/Project_ART into issue#3

Commit:
Cleaned up view page

- Added dummy data
- Loaded user data into table
- Created form queries with get_all_from_gallery

Jessonsotoventura commented on app/controllers/gallery_submission.py:
```diff
@@ -3,14 +3,15 @@
 from flask import session
 
-@app.route('/gallery/submission/<int:gid>', methods=["GET","POST"])
+#@app.route('/gallery/submission/<int:gid>', methods=["GET","POST"])
```
Looks like on the merge you commented this out
This is route that is need

Jessonsotoventura commented on app/controllers/gallery_submission.py:
```diff
@@ -3,14 +3,15 @@
     if(doesUserHaveRole('admin')):
         is_admin = True
-    return render_template('views/gallery_submission.html', gallery=gallery, is_admin=is_admin)
+    return render_template('views/', gallery=gallery, is_admin=is_admin)
```
This new route does not exist, probably got removed during the merge

Jessonsotoventura commented on app/models/Forms.py:
```diff
@@ -15,5 +15,7 @@ class Forms (baseModel):
   cv                = ForeignKeyField(Files, related_name="cv_file")
-  personal_statement = ForeignKeyField(Files, related_name="personal_statment")
+  personal_statement= ForeignKeyField(Files, related_name="personal_statment")
+  submit_date       = DateTimeField()
```
If you add new entires into the model, please update the dummy data file to match

Commit:
Fixed issues with submissions

Jessonsotoventura reviewed (commented):
I have fixed these comments already, but I want you to have a look at the code and review it before it gets merged to development @josezindia

Jessonsotoventura commented:
@josezindia  I made changes to your branch, please have a look and review the changes. 
Comment on anything that is out of place and comment here if you approve the changes

Some comments for future commits:
- Please indent html so that the open and close are on the same line
- Please make sure that if you change the db to change the dummy data file
- Please make sure that when you merge you are in fact updating from development
It looks like the merge was behind, remember to git checkout development and the git pull then rebase..

Commit:
Merge branch 'development' into issue#3

Commit:
Merge branch 'development' into issue#3

Commit:
Fixed issues related to the blueprints merge

Commit:
Fixed issues related to the blueprints merge

Commit:
Merge branch 'issue#3' of github.com:BCStudentSoftwareDevTeam/Project_ART into issue#3
